Fresh

juliaputko opened the pull request.

jklymak reviewed (commented):
This is looking good.  A few minor comments.

hvdosser commented on datapage.html:
```diff
@@ -0,0 +1,70 @@
+
+<h2>Data Description</h2>
+
+The different types of data files that are available are named as follows:
```
Can we change this sentence to "As of 2022, the different types of data files that are available are named as follows: " ?
Then I'll put it back after I catch up on updating the old directories to match the new naming conventions.

hvdosser commented on datapage.html:
```diff
@@ -0,0 +1,70 @@
+
+    <table style="width:100%">
+      <tr>
+        <td><a href="test.pdf" target="_blank">Open PDF</a></td>
```
Let's remove the blank PDFs for now (just comment out if possible), so they don't look like broken links.

hvdosser reviewed (changes requested):
Julia, can you take a look at all of my and Jody's comments, and either reply or make changes as is appropriate? The main concern is whether some of the git pages development code accidentally made it into the pull request. .

hvdosser commented on _includes/head.html:
```diff
@@ -18,17 +18,21 @@
-    <!-- Bootstrap Core CSS -->
-    <link rel="stylesheet" href="{{ "/css/bootstrap.css" | prepend: site.baseurl | prepend: site.url }}">
+      <!-- Bootstrap Core CSS -->
+    <!--link rel="stylesheet" href="{{ "/css/bootstrap.css" | prepend: site.baseurl | prepend: site.url }}">-->
```
It's been re-added now.

jklymak commented on index.html:
```diff
@@ -15,39 +15,94 @@
 <h3> Deployments </h3>
 
-C-PROOF glider deployments are under way!  Click on map to see list of deployments and access data:
-<a href="deployments/">
```
OK< I see - we should add this link back in around the map please...

juliaputko commented:
Hi Jody, 


_includes/head.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838142525>:
I removed site-url because I was having issues loading the css files during my testing, however I tested again just now, and it does not seem to be an issue, so I can revert it. 

_layouts/data_jp.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838142751>
I use this file as a layout for the data.md page. To make the navigation bar, jekyll looks for the word “project” in the front-matter of layouts, so I made a separate layout for this page because I did not want it to appear in the top navigation bar. 

about.m <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838143293>d :
I will review the website you sent, and also talk about it with Hayley. 

data.md <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838144049>
This page is accessible through the ‘wget data’ button on the deployments page. 
As for wgetting the data, this is also something decided by Hayley, so I will need to bring this up with her as well. 

As for changing to raw html in markdown, I made css additions, and as to not clutter the index file with my css, I have added all the css to the bootstrap.css file - however for organizational purposes, and so as to not have to rename everything I added a class so that the css I added could be categorized under that class. (eg. .platforms .p { #css changes } )
> Is there a reason you changed to raw html in a markdown file? That usually shouldn't be needed…"
> 
> <body class="platforms"
> 
> 

Best, 
Julia Putko


> On Mar 29, 2022, at 11:00 PM, Jody Klymak ***@***.***> wrote:
> 
> 
> @jklymak commented on this pull request.
> 
> This is looking good. A few minor comments.
> 
> In _includes/head.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838142525>:
> 
> > @@ -18,17 +18,21 @@
>  
>      <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
>  
> -    <!-- Bootstrap Core CSS -->
> -    <link rel="stylesheet" href="{{ "/css/bootstrap.css" | prepend: site.baseurl | prepend: site.url }}">
> +      <!-- Bootstrap Core CSS -->
> +    <!--link rel="stylesheet" href="{{ "/css/bootstrap.css" | prepend: site.baseurl | prepend: site.url }}">-->
> Shy did we remove site-url here?
> 
> In _layouts/data_jp.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838142751>:
> 
> > @@ -0,0 +1,10 @@
> +---
> Are these two files just test?
> 
> In about.md <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838143293>:
> 
> > +
> +{% for post in site.data.inthenews %}{% if post.visible2 == true %} 
> +<ul class="post-preview">   <!---post-preview  -->
> +    <a href="{{ post.url }}">
> +        <h4 class="post-title">  {{ post.title }}
> +        </h4>
> +    </a>
> +
> +  </ul>
> +
> +{% endif %}
> +{% endfor %}
> +
> +<br>
> +<h3>Publications</h3>
> +
> We are probably going to want a better way to track publications that this. Suggest looking at BibBase. https://bibbase.org <https://bibbase.org/>
> In data.md <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838144049>:
> 
> > @@ -0,0 +1,106 @@
> +---
> Does this page show up anywhere?
> 
> In data.md <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838144518>:
> 
> > +    /* customizable */
> +    background: #979BA0;
> +    margin-top: -35px;
> +  /*  margin-left: 60px;*/
> +}
> +
> +[data-tooltip]:not([data-tooltip-persistent]):before {
> +    pointer-events: none;
> +}
> +
> +</style>
> +
> +# Downloading Glider Data
> +
> +## Setup:
> +
> I wouldn't include install directions for wget here. 1) folks could also use curl, 2) these will go out of date.
> 
> In platforms/index.md <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838146310>:
> 
> >  
> -Under water gliders use a buoyancy engine to make themselves lighter and heavier than the seawater around them, causing them to float or sink.  They have hydrodynamic wings on them that turn some of this lift to forward thrust, allowing them to move through the water at 20-30 km/day.  Every 4-8 hours they surface and communicate home over satellite their GPS position and a subset of their data.  They also accept commands for where to go next, how to change sampling, etc.  
> +
> +<h3>What is an ocean glider? </h3>
> Is there a reason you changed to raw html in a markdown file? That usually shouldn't be needed...
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/c-proof/cproofwebsite/pull/6#pullrequestreview-925554250>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ATOOXH4BCTTN3FKVYILRY2LVCPUZFANCNFSM5R7ZXWNQ>.
> You are receiving this because you authored the thread.

juliaputko commented:
Hi Hayley, 

The files added to .gitignore were added because unless I added them, my installation of jekyll site related things would result in 10,000+ git changes among those related files. 

I changed the name of the class in members.html. jptest was a name that I had chosen early on, and never changed. It is changed to people_ now. 

cproof-deployments_argo.geojson is now cproof-deployments_all.geojson 

I changed that one line in the description on the data page. 

The blank pdfs are commented out. 


I will make a new commit with all those changes, it should just append itself to the last pull request. 


Best, 
Julia Putko

> On Apr 25, 2022, at 10:08 AM, hvdosser ***@***.***> wrote:
> 
> 
> @hvdosser requested changes on this pull request.
> 
> Julia, can you take a look at all of my and Jody's comments, and either reply or make changes as is appropriate? The main concern is whether some of the git pages development code accidentally made it into the pull request. .
> 
> In .gitignore <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r857806822>:
> 
> > @@ -1,3 +1,8 @@
>  _site
>  .sass-cache
>  .jekyll-metadata
> +
> +.bundle/
> +gliderdata/
> +vendor/bundle/ruby/3.1.0/
> Were these changes specific to git pages, or do we need them for the main website too?
> 
> In members.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r857824573>:
> 
> > @@ -32,32 +79,96 @@ <h3>{{ member.name }}</h3>
>  <span class="social-share-googleplus"><a href="{{ member.gplus }}" title="Google Plus"><img src="{{ "/img/icons/gplus-icon.png" | prepend: site.baseurl }}" style="height:20px">Google Plus</a></span>
>  {% endif %}
>  <br/>
> +<p class="jptest"></p>
> Is "jptest" the right class for the live website or was it only for dev?
> 
> In js/map.js <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r857826453>:
> 
> > @@ -23,7 +23,7 @@ var featureLayer = L.geoJson(null, {
>  });
>  
>  // Fetch the GeoJSON file
> -$.getJSON("/gliderdata/deployments/cproof-deployments.geojson", function (data) {
> +$.getJSON("/gliderdata/deployments/cproof-deployments_argo.geojson", function (data) {
> cproof-deployments_argo includes the ocean gliders and argo, right? It might be better to name it "cproof-deployments-all" instead, to avoid confusion.
> 
> In datapage.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r857846520>:
> 
> > +<table >
> +  <tr>
> +    <td>
> +      <p>
> +        <a href="{{ site.baseurl }}/data.html"><strong>Direct download</strong></a>: Wget the data for all missions, by glider line, or by platform </br>
> +        <a href='{{ site.baseurl }}/deployments/index.html'><strong>Deployments</strong></a>: View and download data and figures for individual glider missions
> +      </p>
> +    </td>
> +  </tr>
> + 
> +</table>
> +
> +
> +<h2>Data Description</h2>
> +
> +The different types of data files that are available are named as follows:
> Can we change this sentence to "As of 2022, the different types of data files that are available are named as follows: " ?
> Then I'll put it back after I catch up on updating the old directories to match the new naming conventions.
> 
> In datapage.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r857846823>:
> 
> > +<ul>
> +    <li>Realtime: Subset of data transmitted via Iridium while the glider is in mission, with basic automated processing steps and QA/QC applied. </li>
> +    <li>Delayed: Full dataset collected by the glider and recovered post-mission, with all processing steps and QA/QC applied. </li>
> +    <li>Level 0 (L0) data: Merged raw data files, with no interpolation and no processing. </li>
> +    <li>Level 1 (L1) data: Time series or profile data with some interpolation and calculation of additional variables like salinity. </li>
> +    <li>Level 2 (L2) data: Gridded level 1 data, on a depth-time grid. </li>
> +</ul>
> +
> +
> +
> +
> +    <h2>Data Processing  </h2>
> +
> +    <table style="width:100%">
> +      <tr>
> +        <td><a href="test.pdf" target="_blank">Open PDF</a></td>
> Let's remove the blank PDFs for now (just comment out if possible), so they don't look like broken links.
> 
> In about.md <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r857830697>:
> 
> > +
> +{% for post in site.data.inthenews %}{% if post.visible2 == true %} 
> +<ul class="post-preview">   <!---post-preview  -->
> +    <a href="{{ post.url }}">
> +        <h4 class="post-title">  {{ post.title }}
> +        </h4>
> +    </a>
> +
> +  </ul>
> +
> +{% endif %}
> +{% endfor %}
> +
> +<br>
> +<h3>Publications</h3>
> +
> Agreed. I will approve for now, and look into BibBase later on.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/c-proof/cproofwebsite/pull/6#pullrequestreview-952137179>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ATOOXH4IF3XGZVFN2AXN3GDVG3GPRANCNFSM5R7ZXWNQ>.
> You are receiving this because you authored the thread.
